gcm_filters/filter.py

```diff
@@ -369,11 +387,14 @@ def plot_shape(self, ax=None):
 
     def apply(self, field, dims):
         """Filter a field with scalar Laplacian across the dimensions specified by dims."""
-        if not issubclass(self.Laplacian, BaseScalarLaplacian):
+        if issubclass(self.Laplacian, BaseVectorLaplacian):
             raise ValueError(
                 f"Provided Laplacian {self.Laplacian} is a vector Laplacian. "
                 f"The ``.apply`` method is only suitable for scalar Laplacians."
             )
+        if issubclass(self.Laplacian, BaseScalarLaplacianWithArea):
+            # simple fixed factor filtering multiplies field by area before filtering
+            field = field * self.grid_ds["area"]
```

Comment:
Rather than putting this login in the `filter` module, why not put it into the kernel itself?